loader/windows/icd_windows_hkr.c

```diff
@@ -131,8 +131,15 @@ static bool ReadOpenCLKey(DEVINST dnDevNode)
 
         if (REG_SZ != dwLibraryNameType)
```

Comment:
Cosmetic: we could simplify the code below (and thereby decrease the indentation level) by:
if (REG_SZ != dwLibraryNameType && REG_MULTI_SZ == dwLibraryNameType)
{
    KHR_ICD_TRACE("Accepting multi-string registry key type\n");
}
else
{
    KHR_ICD_TRACE("Unexpected registry entry 0x%x! continuing\n", dwLibraryNameType);
}

Reply:
This logic isn't equivalent. It would catch the REG_SZ key type as an invalid registry entry.  I'll keep the block as is, it makes it clear to the reader than the REG_MULTI_SZ is considered to be an anomaly.

Reply:
You're right, my bad.